File: CareConnect/CareConnect.Models/SearchObjects/ClientAdditionalData.cs
using CareConnect.Models.SearchObjects;

namespace CareConnect.Models.SearchObjects
{
    public class ClientAdditionalData : BaseAdditionalSearchRequestData
    {
        public bool? IsUserIncluded { get; set; }

        //public bool? IsChildrenIncluded { get; set; }
    }
}

File: UI/careconnect_admin/lib/models/search_objects/client_additional_data.dart
import 'package:json_annotation/json_annotation.dart';

part 'client_additional_data.g.dart';

@JsonSerializable(explicitToJson: true)
class ClientAdditionalData {
  bool? isUserIncluded;
  bool? isChildrenIncluded;

  ClientAdditionalData({this.isUserIncluded, this.isChildrenIncluded});

  factory ClientAdditionalData.fromJson(Map<String, dynamic> json) =>
      _$ClientAdditionalDataFromJson(json);

  Map<String, dynamic> toJson() => _$ClientAdditionalDataToJson(this);
}
